METRICS.md

```diff
@@ -0,0 +1,88 @@
+## Collected Metrics
+
+This plugin collects the raw counters and a few other metrics from etcd.  It is focused on the ones which are easily parsable and excludes ones which are formatted specifically for a Prometheus metric.  There are also a handful of derivatives which take a sum over count average.
+
+Namespace | Data Type | Description
+----------|-----------|-----------------------
+/intel/etcd/derivative/etcd_server_proposal_durations_avg | | 
+/intel/etcd/derivative/etcd_snapshot_save_marshalling_durations_seconds_avg | | 
```

Comment:
Is there plan to add description and data type as well?

cc: @Armatorix

Reply:
This metrics aren't describes in etcd documentation, so it may be hard to add them. Data types are unknown as well, even to Snap.